activerecord/test/cases/adapters/postgresql/comment_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'cases/helper'
+require 'support/schema_dumping_helper'
+
+class PostgresqlCommentTest < ActiveRecord::PostgreSQLTestCase
```

Comment:
This looks a *lot* like the MySQL equivalent. Do we have precedent for that? Shouldn't this be a generic test, which just gets skipped on sqlite3?